docs/getting_started/getting_started.md

```diff
@@ -2,14 +2,26 @@
 
 In this section, we will cover the basics of getting started with installing libhal on your machine and flashing the Astraeus-I board. We will also cover how to create your own project.
 
+### <img src="../../assets/libhal.png" style="width: 30px; height: auto;"> Libhal Introduction
+
+Libhal is a software library designed to abstract complexities associated with interfacing with different hardware platforms. By offering a unified API, libhal enables developers to write software that's independent of the underlying hardware specifics. This allows for seamless portability across different platforms, ensuring that the same code can be used to run on different processors, mircrocontrollers, systems, and devices. Libhal is designed to be modular and extensible. This allows developers to add support for new hardware platforms and devices.
+
+To learn more on the installation process, please visit the <a href="https://libhal.github.io/2.2/getting_started/" target="_blank">libhal docs</a>
+ to learn how to install and configure libhal on your machine, including all the prerequities below . Make sure you go through all of the steps in the docs before continuing. This includes installing and flashing to your Astraeus-I board.
+
```

Comment:
This looks good but it should be under the dependencies because our board is dependant on libhal and its drivers.